README.md

```diff
@@ -1 +1,94 @@
-# federated-collection-discovery-deployment
\ No newline at end of file
+# Federated Collection Discovery Deployment
+
+## About
+
+This is a deployment repo for MAAP's instance of the
```

Comment:
Could make the ds repository into a template construct repo, then pass in maap configurations but maybe that's for another day 🤪

Reply:
Yeah it would make more sense to have all of the CDK logic over in the DS repo for a few reasons. We could publish it like we do with `eoapi-cdk` and just import it where we want to deploy it. If we want to re-use this deployment somewhere else I'll probably do that!